### PR TITLE
Cpu mode fixes and openvino export with input_format

### PR DESCRIPTION
**Pull request by Ilya-Krylov**
* get_flops fix for cpu mode
* batch size selection workaround for cpu mode
* openvino export with input_format parameter

### CODE REVIEW

File: tools/get_flops.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def main():
     if args.update_config is not None:
         cfg.merge_from_dict(args.update_config)
     model = build_detector(
-        cfg.model, train_cfg=cfg.train_cfg, test_cfg=cfg.test_cfg).cuda()
```
Comment (druzhkov-paul) [inline tools/get_flops.py:39]: I think that CUDA should be used when GPUs are available, because some models do contain CUDA-only primitives.

Comment (druzhkov-paul) [reply tools/get_flops.py:39]: Does it make sense to do the same thing for export script? Could you please implement this as well?

**Comment by Ilya-Krylov** [comment]
run tests

File: tools/get_flops.py (hunk 1 above)
Comment (Ilya-Krylov) [reply tools/get_flops.py:39]: added

**Review by druzhkov-paul — APPROVED** [review]

**Pull request merged**